src/Bundle/Resources/config/services.xml

```diff
@@ -13,13 +13,12 @@
             <argument type="service" id="serializer.mapping.class_discriminator_resolver" on-invalid="ignore" />
         </service>
 
-        <service id="dunglas_doctrine_json_odm.normalizer.object" class="Dunglas\DoctrineJsonOdm\Normalizer\ObjectNormalizer" public="false">
-            <argument type="service" id="dunglas_doctrine_json_odm.normalizer.raw_object" />
-        </service>
+        <service id="dunglas_doctrine_json_odm.normalizer.array" class="Symfony\Component\Serializer\Normalizer\ArrayDenormalizer" public="false" />
 
-        <service id="dunglas_doctrine_json_odm.serializer" class="Symfony\Component\Serializer\Serializer" public="true">
+        <service id="dunglas_doctrine_json_odm.serializer" class="Dunglas\DoctrineJsonOdm\Serializer" public="true">
             <argument type="collection">
-                <argument type="service" id="dunglas_doctrine_json_odm.normalizer.object" />
+                <argument type="service" id="dunglas_doctrine_json_odm.normalizer.array" />
```

Comment:
Instead of defining custom ones, I'd like to use standard symfony normalizers here

-     `serializer.denormalizer.array`
-     `serializer.normalizer.object`

@Toflar WDYT?

Reply:
I guess we could do that, I think @dunglas did not do that to just make sure it keeps working whatever Symfony decides to do with the service id's and aliases.

Reply:
OK, let's wait for @dunglas feedback

Reply:
LGTM

Reply:
Okay @ajgarlag, would you update this? I'll then merge and release as `1.0.0-rc1` so we can go for some final tests before the stable release.